modal/_container_entrypoint.py

```diff
@@ -2,6 +2,8 @@
 # ruff: noqa: E402
 import os
 
+from modal._clustered_functions import initialize_clustered_function
```

Comment:
Is there a reason we do this above all the other Modal imports?

Reply:
no, thanks for catching this!